torchaudio/models/wav2vec2/pretrained.py

```diff
@@ -122,6 +122,530 @@ def _get_labels():
+        "encoder_layer_drop": 0.2,
+        "aux_num_out": 32,
+    },
+    _labels=_get_labels(),
+)
+WAV2VEC2_ASR_LARGE_960H.__doc__ = """Build "large" wav2vec2 model with an extra linear module
+
+Pre-trained and fine-tuned for ASR on 960 hours of
+*LibriSpeech* [:footcite:`7178964`] dataset.
+
+Originally published by the authors of *wav2vec 2.0*
+[:footcite:`baevski2020wav2vec`].
+[`Source <https://github.com/pytorch/fairseq/tree/main/examples/wav2vec#pre-trained-models>`__]
+"""
+
+WAV2VEC2_LARGE_LV60K = Wav2Vec2PretrainedModelBundle(
+    'wav2vec2_fairseq_large_lv60k.pth',
+    {
+        "extractor_mode": "layer_norm",
+        "extractor_conv_layer_config": [
+            (512, 10, 5),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 2, 2),
+            (512, 2, 2),
+        ],
+        "extractor_conv_bias": True,
+        "encoder_embed_dim": 1024,
+        "encoder_projection_dropout": 0.1,
+        "encoder_pos_conv_kernel": 128,
+        "encoder_pos_conv_groups": 16,
+        "encoder_num_layers": 24,
+        "encoder_num_heads": 16,
+        "encoder_attention_dropout": 0.1,
+        "encoder_ff_interm_features": 4096,
+        "encoder_ff_interm_dropout": 0.0,
+        "encoder_dropout": 0.0,
+        "encoder_layer_norm_first": True,
+        "encoder_layer_drop": 0.0,
+        "aux_num_out": None,
+    },
+    _labels=None,
+)
+WAV2VEC2_LARGE_LV60K.__doc__ = """Build "large-lv60k" wav2vec2 model.
+
+Trained on 60,000 hours of *LibriLight* [:footcite:`librilight`] dataset. Not fine-tuned.
+
+Originally published by the authors of *wav2vec 2.0*
+[:footcite:`baevski2020wav2vec`].
+[`Source <https://github.com/pytorch/fairseq/tree/main/examples/wav2vec#pre-trained-models>`__]
+"""
+
+WAV2VEC2_ASR_LARGE_LV60K_10M = Wav2Vec2PretrainedModelBundle(
+    'wav2vec2_fairseq_large_lv60k_asr_ll10m.pth',
+    {
+        "extractor_mode": "layer_norm",
+        "extractor_conv_layer_config": [
+            (512, 10, 5),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 2, 2),
+            (512, 2, 2),
+        ],
+        "extractor_conv_bias": True,
+        "encoder_embed_dim": 1024,
+        "encoder_projection_dropout": 0.1,
+        "encoder_pos_conv_kernel": 128,
+        "encoder_pos_conv_groups": 16,
+        "encoder_num_layers": 24,
+        "encoder_num_heads": 16,
+        "encoder_attention_dropout": 0.1,
+        "encoder_ff_interm_features": 4096,
+        "encoder_ff_interm_dropout": 0.0,
+        "encoder_dropout": 0.0,
+        "encoder_layer_norm_first": True,
+        "encoder_layer_drop": 0.0,
+        "aux_num_out": 32,
+    },
+    _labels=_get_labels(),
+)
+WAV2VEC2_ASR_LARGE_LV60K_10M.__doc__ = """Build "large-lv60k" wav2vec2 model with an extra linear module
+
+Pre-trained on 60,000 hours of *Libri-Light* [:footcite:`librilight`] dataset, and
```

Comment:
From [this table](https://github.com/pytorch/fairseq/tree/main/examples/wav2vec#pre-trained-models) and your `WAV2VEC2_ASR_LARGE_LV60K_100H` doc below, I think this should be fine-tuned on LibriSpeech instead of Libri-Light

Reply:
Thanks for spotting the error. I looked at the paper again and it turned out that `LibriVox` is the correct one.

The following is the relationship between these datasets.
- `LibriVox` (60,000 hours audio)
  - `LibriSpeech` (960 hours audio + transcript, subset of `LibriVox`)
    - `LibriLight` (subset of `LibriSpeech` training subset)